src/components/screens/AddonScreen/AddonScreen.js

```diff
@@ -290,6 +290,11 @@ export function PureAddonScreen({ data: { allMediumPost }, ...props }) {
           desc="Select between JSS themes."
           addonUrl="https://github.com/vertexbz/storybook-addon-jss-theme"
         />
+        <AddonItem
+          title="Root Attribute"
+          desc="Storybook addon, which provide ability to change html or body attribute."
```

Comment:
I'd suggest simplifying the desc text –it's listed as an addon so calling it a "Storybook addon" is not needed. How about this:
```suggestion
          desc="Provides the ability to change the html or body attribute."
```